Gemfile

```diff
@@ -83,8 +83,13 @@ else
 end
 
 if RUBY_VERSION >= '2.4.0'
-  gem 'rubocop', ['~> 1.10', '< 1.33.0'], require: false
-  gem 'rubocop-packaging', '~> 0.5', require: false
+  if RUBY_VERSION >= '2.6.0'
+    gem 'rubocop', '~> 1.34.0', require: false
+    gem 'rubocop-packaging', '~> 0.5.2', require: false
```

Comment:
We need a higher version of `rubocop-packaing` because the library uses a deprecated method, so it is annoying to see those messages on your terminal. 

Looks like @marcotc fixed upstream https://github.com/utkarsh2102/rubocop-packaging/pull/43 😉

Reply:
To be honest, I'm not sure it's worth keeping the alternative version for 2.4/2.5, given that anyway it'll probably not work as we may start referencing new cops and whatnot in the rubocop configuration.

Loic actually suggested something like this in #2206, which was the last time someone poked this.

Thus, consider making the `if RUBY_VERSION >= 2.6.0` the top-level check and drop the part for 2.4/2.5 :)

Reply:
Changed https://github.com/DataDog/dd-trace-rb/pull/2870/commits/8ed2408c5d1ed864a4a270030d720167b4d9af5c